Cargo.toml

```diff
@@ -22,6 +22,11 @@ lto = "thin"
 mbedtls = { git = "https://github.com/mobilecoinfoundation/rust-mbedtls.git", rev = "6d8fe323a3292f87a6bce4b35963d47139a583f9" }
 mbedtls-sys-auto = { git = "https://github.com/mobilecoinfoundation/rust-mbedtls.git", rev = "6d8fe323a3292f87a6bce4b35963d47139a583f9" }
 
+mc-sgx-dcap-types = { git = "https://github.com/mobilecoinfoundation/sgx.git", rev = "b86126432d5e5a7705bb7be9d3358963910f2ccc" }
```

Comment:
These will need to go away prior to this merging, waiting on https://github.com/mobilecoinfoundation/sgx/pull/417

Reply:
removed now